.github/workflows/build.yml

```diff
@@ -162,6 +160,18 @@ jobs:
           curl -sLO https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable/openshift-client-linux.tar.gz
           tar zxvf openshift-client-linux.tar.gz oc
 
+      - name: Install chart-verifier
+        id: install-chart-verifier
+        if: ${{ steps.check_build_required.outputs.run-build == 'true' }}
+        uses: redhat-actions/openshift-tools-installer@v1
+        with:
+          # Search through these projects' GitHub releases, instead of the OpenShift Mirror.
+          # https://docs.github.com/en/actions/reference/authentication-in-a-workflow
+          source: "github"
+          # Using GitHub token from the github context
+          github_pat: ${{ secrets.GITHUB_TOKEN }}
+          chart-verifier: "latest"
```

Comment:
For development we need to use "main" which unfortunately does not have a binary. Don't think we can use it in this form. A test of the github action is better in the chart verifier builds as a separate workflow. 
Also we do not want the chart-certifcation flow as a testing ground for the action.